resources/shared-db/Cargo.toml

```diff
@@ -14,5 +14,13 @@ shuttle-service = { path = "../../service", version = "0.36.0" }
 sqlx = { version = "0.7.1", optional = true }
 
 [features]
-postgres = ["sqlx/postgres", "sqlx/runtime-tokio-native-tls"]
-postgres-rustls = ["sqlx/postgres", "sqlx/runtime-tokio-rustls"]
+default = []
+
+# Mongodb
+mongodb = ["dep:mongodb"]
```

Comment:
This line should not be needed right? The dependency is already optional (which implicitly adds this line)

Reply:
Yes, but I prefer having it this way since it makes the list of features much easier to read. (cargo feature docs are usually hard to find, and I often find myself looking up the manifest file)

Reply:
Hmm, I understand. Don't think it is idiomatic Rusty though. Will personally prefer not to have it.

Reply:
An explicit list of features with comments is IMO always better than looking for the optional tags. 
Examples of explicit feature->dep mapping with docs: 
https://github.com/emilk/egui/blob/7733d1d87c31849d4ac8a3039131c8ad3e71b547/crates/epaint/Cargo.toml#L61
https://github.com/emilk/egui/blob/7733d1d87c31849d4ac8a3039131c8ad3e71b547/crates/egui/Cargo.toml#L27
Not sure what the idiom is tho :)